package.json

```diff
@@ -29,6 +29,7 @@
     "@rhinestone/checknsignatures": "github:rhinestonewtf/checknsignatures",
     "@rhinestone/flatbytes": "github:rhinestonewtf/flatbytes",
     "@rhinestone/modulekit": "^0.4.9",
+    "@rhinestone/module-bases": "github:rhinestonewtf/module-bases#fix/remediations",
```

Comment:
we'll be able to remove this when we merge this branch into module-bases right

Reply:
correct